news/2024/2024-01-28-community-choice-2023-open.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+title: "Community Choice 2023: Voting Open"
+date: 2024-01-28 00:00:00 +0000
+---
+
+A new year dawns upon us, but let's take one last step back to crown the ten most liked maps of 2023 today!
+
+![](https://assets.ppy.sh/contests/195/header.jpg)
+
+It's that time of the year again, where circle clickers, drum bashers, fruit catchers, and key smashers all gather together in a giant circle to ferociously debate which maps deserve to be recognised as the best of the best, in a trend as old as time itself (which some would call the "Best Of" polls in ye olden days).
+
+To be real, the giant circle meeting may not be a thing, but the actual voting definitely is. You have **10 votes per game mode** to cast on **any map ranked during 2023 that you have played at least once**. Each game mode's contest listing will automagically conjure up all the eligible maps that you can vote in.
+
+The top 10 picks will be dubbed as the *Community Favourites of 2023* as usual, with the #1 from each game mode bestowed a prestigious new profile badge and 6 months of osu!supporter, alongside 1 month of osu!supporter for each of the top 10 finalists.
+
+As per usual, we'll also be running the **Mappers' Favourite** award for each game mode, where we take the votes of anyone who has ranked at least one map ever into a distinct award to give the mapping community a proper say in what beatmaps really defined the scene last year. This all happens automatically, so make sure you cast your votes if you want to make sure your favourite maps have their justice.
+
+Before we proceed, let's go through a few mapping trends from the past year:
+
+- A total of **17,342** difficulties were ranked across **4,665** mapsets in all game modes, seeing roughly a **13%** drop from last year.
+- **8,917** osu! difficulties were blessed with nominations from 2 Beatmap Nominators, with a **20%** decrease from 2022.
+- **3,345** osu!taiko difficulties successfully made it through their ranking journey, returning a surprising increase of **23%** from last year.
+- **1,050** osu!catch difficulties were deemed worthy to receive permanent leaderboards, matched with a **26%** drop from 2022.
+- **4,030** osu!mania difficulties got recognised as enough high quality to have an everlasting place in the Ranked section, with an astonishing increase of **34%** from last year to continue their upward trend!
+
+The overall pace with ranking maps may have slowed down a bit compared to the past few years, but it's still going strong and steady!
```

Comment:
```suggestion
The astute among you may observe a discrepancy between these numbers and [last year](https://osu.ppy.sh/home/news/2023-01-24-community-choice-2022-open)'s, but worry not, because we've simply excluded deleted maps from calculations this time.

With that said, the overall pace of ranking maps may have slowed down a bit compared to the past few years, but it's still going strong and steady!
```